docs/src/process.rst

```diff
@@ -111,10 +111,38 @@ Data types
     ::
 
         typedef enum {
+            /*
+            * The following four options are mutually-exclusive, and define
+            * the operation to perform for the corresponding file descriptor
+            * in the child process:
+            */
+
+            /*
+            * No file descriptor will be provided (or redirected to
+            * `/dev/null` if it is fd 0, 1 or 2).
+            */
             UV_IGNORE = 0x00,
+
+            /*
+            * TODO(leonerd): I imagine this creates a pipe of some kind,
+            * giving one end to the created process. I have no idea what
+            * happens to the other end.
+            /*
```

Comment:
That's right. The child process inherits the other end it in the right file descriptor position.